lib/diagnostic.rb

```diff
@@ -115,10 +120,10 @@
 # Use shorthand to make this one line long.
 # Write your response as a Ruby string.
 
-Diagnostic.person_hash = 'your answer here'
+Diagnostic.person_hash = 'person = Hash.new().merge({ :age=> 22, :first_name=> 'Shireen'})'
```

Comment:
Yes, but an easier way would be 
`Diagnostic.person_hash = person { :age=> 22, :first_name=> 'Shireen'}` OR `Diagnostic.person_hash = person { age: 22, first_name: 'Shireen'}`